### PR TITLE
ie/iee: remove baddr ##bin

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/cbin.c [hunk 1]
```diff
@@ -1092,11 +1092,14 @@ static int bin_entry(RCore *r, int mode, ut64 laddr, int va, bool inifin) {
 		r_cons_printf ("[");
 	} else if (IS_MODE_NORMAL (mode)) {
 		if (inifin) {
-			r_cons_printf ("[Constructors]\n");
+			r_cons_printf ("[Constructors] ");
 		} else {
-			r_cons_printf ("[Entrypoints]\n");
+			r_cons_printf ("[Entrypoints] ");
```
Comment (radare) [inline libr/core/cbin.c]: that extra space should go in the baddr line below not here

File: libr/core/cbin.c [hunk 2]
```diff
@@ -1175,9 +1178,8 @@ static int bin_entry(RCore *r, int mode, ut64 laddr, int va, bool inifin) {
 			r_cons_printf (
 				 "vaddr=0x%08"PFMT64x
```
Comment (radare) [inline libr/core/cbin.c]: and h ere

File: libr/core/cbin.c (hunk 1 above)
Comment (kazarmy) [reply libr/core/cbin.c]: emm ok 789ccb60af28fb60f0b4e397c0db7a307d773e62

**Comment by ret2libc** [comment]
Mmm.. why are we removing that `baddr` info? Don't we need it anymore?

**Comment by kazarmy** [comment]
We can get it from `iI~baddr` and it's being repeated for every entry line.

**Pull request merged**